package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "autolabcling",
```

Comment:
The name should be `autolabjs`. This is to bring consistency in the names across all the projects.